agent/04_agent_configure.sh

```diff
@@ -60,6 +65,20 @@ function get_static_ips_and_macs() {
     done
 }
 
+function setNetworkingVars() {
+   if [[ "$IP_STACK" = "v4" ]]; then
+    CLUSTER_NETWORK=${CLUSTER_SUBNET_V4}
+    SERVICE_NETWORK=${SERVICE_SUBNET_V4}
+    MACHINE_NETWORK=${EXTERNAL_SUBNET_V4}
+    CLUSTER_HOST_PREFIX=${CLUSTER_HOST_PREFIX_V4}
+  elif [[ "$IP_STACK" = "v6" ]]; then
+    CLUSTER_NETWORK=${CLUSTER_SUBNET_V6}
+    SERVICE_NETWORK=${SERVICE_SUBNET_V6}
+    MACHINE_NETWORK=${EXTERNAL_SUBNET_V6}
+    CLUSTER_HOST_PREFIX=${CLUSTER_HOST_PREFIX_V6}
+  fi
+}
```

Comment:
Why do we have to set these values again here?

Reply:
ok, so we can set it always irrespective of networking mode. 
https://github.com/pawanpinjarkar/dev-scripts/blob/c1716abad59ec5e2684677410d3129a38bdfc83b/agent/04_agent_configure.sh#L379-L383

Reply:
Maybe it was added initially to help with better readability?

Reply:
Sorry if I wasn't clear in my first comment. This is duplicate code and should be removed. Network settings are already set up in [network.sh](https://github.com/openshift-metal3/dev-scripts/blob/master/network.sh#L86-L134)

Reply:
There are needed to set up the subnets to use in the manifests based on the IP_STACK and the settings in network.sh, but these really should be lowercase as they're not env variables and not used outside of this file